diff --git a/test/msgpack_basic.cpp b/test/msgpack_basic.cpp
index 32e9c493..70de5c2d 100644
--- a/test/msgpack_basic.cpp
+++ b/test/msgpack_basic.cpp
@@ -209,7 +209,7 @@ struct TypePair {
 template <typename T>
 class IntegerToFloatingPointTest : public testing::Test {
 };
-TYPED_TEST_CASE_P(IntegerToFloatingPointTest);
+TYPED_TEST_SUITE_P(IntegerToFloatingPointTest);

 TYPED_TEST_P(IntegerToFloatingPointTest, simple_buffer)
 {
@@ -234,14 +234,14 @@ TYPED_TEST_P(IntegerToFloatingPointTest, simple_buffer)
     }
 }

-REGISTER_TYPED_TEST_CASE_P(IntegerToFloatingPointTest,
+REGISTER_TYPED_TEST_SUITE_P(IntegerToFloatingPointTest,
                            simple_buffer);

 typedef testing::Types<TypePair<float, signed long long>,
                        TypePair<float, unsigned long long>,
                        TypePair<double, signed long long>,
                        TypePair<double, unsigned long long> > IntegerToFloatingPointTestTypes;
-INSTANTIATE_TYPED_TEST_CASE_P(IntegerToFloatingPointTestInstance,
+INSTANTIATE_TYPED_TEST_SUITE_P(IntegerToFloatingPointTestInstance,
                               IntegerToFloatingPointTest,
                               IntegerToFloatingPointTestTypes);
